docs/operator.md

```diff
@@ -71,6 +71,14 @@ Enables or disables [automatic storage scaling / volume expansion](scaling.md#au
 | ----------- | ---------- |
 | :material-toggle-switch-outline: boolean     | `false`  |
 
+### `enableExternalVolumeAutoscaling`
+
+Enables or disables the use of external volume autoscaler. 
```

Comment:
Maybe we can add also this sentence. When disabled, the operator-managed volume expansion can be used through the  `enableVolumeExpansion` configuration.